flow/scene_update_context.cc

```diff
@@ -66,18 +66,18 @@ void SetMaterialColor(scenic::Material& material,
 }  // namespace
 
 SceneUpdateContext::SceneUpdateContext(std::string debug_label,
+                                       scenic::Session* session,
                                        fuchsia::ui::views::ViewToken view_token,
                                        scenic::ViewRefPair view_ref_pair,
-                                       SessionWrapper& session,
                                        bool intercept_all_input)
     : session_(session),
-      root_view_(session_.get(),
+      root_view_(session_,
                  std::move(view_token),
                  std::move(view_ref_pair.control_ref),
                  std::move(view_ref_pair.view_ref),
-                 debug_label),
-      metrics_node_(session.get()),
-      layer_tree_node_(session_.get()) {
+                 std::move(debug_label)),
+      metrics_node_(session_),
+      layer_tree_node_(session_) {
```

Comment:
I'm used to seeing at least a DCHECK() for a raw pointer passed in.

Reply:
Will add DCHECK